http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoMetricsMonitor.scala

```diff
@@ -115,7 +115,8 @@ class LeoMetricsMonitor[F[_]](config: LeoMetricsMonitorConfig,
       c <- r.images.filter(i => imageTypes.contains(i.imageType)).headOption
     } yield Map(
       RuntimeStatusMetric(r.cloudContext.cloudProvider,
-                          c,
+                          c.imageType,
```

Comment:
This is an unrelated metrics fix, which I also happened to notice since the release on 3/28. Storing `RuntimeImage` in the `RuntimeStatusMetric` and `RuntimeHealthMetric` case classes was breaking aggregation because it holds a timestamp field. Changed that type to just contain the fields we care about for metrics (imageType and imageUrl).